db/migrate/20220405041748_contributions_not_null.rb

```diff
@@ -0,0 +1,6 @@
+class ContributionsNotNull < ActiveRecord::Migration[6.1]
+  def change
+    change_column_null(:contributions, :author_id, false)
+    change_column_null(:contributions, :publication_id, false)
+  end
```

Comment:
we will allow cap_profile_id to continue to be null since some older records have it as null and the rest of the app seems to work fine with it being null as well